recipes/harfbuzz/all/conanfile.py

```diff
@@ -92,48 +95,81 @@ def requirements(self):
         if self.options.with_icu:
             self.requires("icu/71.1")
         if self.options.with_glib:
-            self.requires("glib/2.73.3")
+            self.requires("glib/2.74.1")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def generate(self):
+        def is_enabled(value):
+            return "enabled" if value else "disabled"
+
+        def meson_backend_and_flags():
+            if is_msvc(self) and Version(self.settings.compiler.version) == "15" and self.settings.build_type == "Debug":
```

Comment:
This compiler version only corresponds to `Visual Studio` and not `msvc`. Can you add a check for `msvc`, too? We don't want this recipe to break for anyone using `msvc`.